swift/aperture/Recorder.swift

```diff
@@ -36,7 +36,10 @@ public class Recorder: NSObject, AVCaptureFileOutputRecordingDelegate {
     }
 
     self.output = AVCaptureMovieFileOutput();
-    self.output?.movieFragmentInterval = CMTimeMake(1,1); // write data to file every 1 second
+
+    // write data to file every 2 seconds
+    // writing every 1 second causes intermittent errors
+    self.output?.movieFragmentInterval = CMTimeMakeWithSeconds(2, 1);
```

Comment:
This is the only way we're in control of the buffering unfortunately. We could go even more low-level, but that would be a lot of work for minimal gain.